setup.py

```diff
@@ -0,0 +1,30 @@
+from os.path import realpath, dirname, join
+from setuptools import setup, find_packages
+
+DISTNAME = 'leelastar'
+DESCRIPTION = 'LeelaStar: Crowd-Sourced Deep Reinforcement Learning'
```

Comment:
Are we going the general framework route? There's quite a few good solutions out there, so I think focusing on a niche of 1v1 StarCraft 2 would make more sense.

Reply:
I am open to new suggestions.

What about “LeelaStar: Crowd-Sourced Deep Reinforcement Learning Agent for Starcraft 2”?